### ISSUE TITLE
do you want to promote this project more?

### ISSUE THREAD

**Issue by lasomethingsomething** [zalando/exhibitor-appliance#14]
Do you think this will be useful to non-Zalandos? LMK, and I'll add it to my growing list of README to-do's!

**Comment by CyberDem0n** [comment]
This project just packages Netflix Exhibitor (https://github.com/Netflix/exhibitor/) and Zookeeper into Docker and Stups.
I think outside of Zalando this project is mostly useless.

**Comment by drummerwolli** [comment]
i would rephrase the last sentence: outside of the STUPS Toolbox this is mostly useless and needs no additional promotion (I don't know how far the efforts have come to bring STUPS to other companies).
